node-tests/blueprints/component-test.js

```diff
@@ -324,6 +329,21 @@ describe('Blueprint: component', function () {
       );
     });
 
+    it('component foo --strict', function () {
+      return emberGenerateDestroy(['component', 'foo', '--strict'], (_file) => {
+        expect(_file('app/components/foo.gjs')).to.equal(templateTagContents);
```

Comment:
This is currently failing for 

```
yarn test:blueprints:ts
```

If someone could point me in the direction where TypeScript test files are converted to js that would be helpful. I suspect it's not handling `.gts` yet.

Reply:
https://github.com/ember-cli/ember-cli/blob/master/lib/models/blueprint.js#L476 maybe?

Reply:
The logic being in a different repository would explain a lot 😅 

Thanks for sharing, I'll fiddle around with that and see if I can get it working.

Reply:
Fixed upstream in ember-cli: https://github.com/ember-cli/ember-cli/pull/10332